ui/seek_bar.js

```diff
@@ -661,22 +673,25 @@ shaka.ui.SeekBar = class extends shaka.ui.RangeElement {
       if (nextChapters.length && (languageChanged || chaptersChanged)) {
         language = nextLanguage;
         chapters = nextChapters;
-        if (abortController) {
-          abortController.abort();
-        }
-        abortController = new AbortController();
-        this.createChapterElements_(
-            this.container, chapters, abortController.signal);
+
+        this.createChapterElements_(this.container, chapters);
+      } else if (!nextChapters.length) {
```

Comment:
Actually, doesn't this mean that if there are no new chapters, the `language` and `chapters` values will stay unchanged? And therefore, if we do go from having chapters to not having chapters, this will continue to think we are still playing whatever language we were before?

If that's not the desired behavior, it might be better to do it like this:
```js
language = nextLanguage;
chapters = nextChapters;
if (!nextChapters.length) {
  this.deletePreviousChapters_();
} else if (languageChanged || chaptersChanged) {
  this.createChapterElements_(this.container, chapters);
}
```

Reply:
Changed!